package.json

```diff
@@ -24,8 +24,9 @@
     "monaco-editor": "^0.52.2",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-icons": "^5.4.0",
-    "typescript": "^5.7.2"
```

Comment:
we need to solve the conflicts by restoring version of react-icons and typescript from `main`